src/engine/subscription/Subscription.ts

```diff
@@ -0,0 +1,311 @@
+import { uniq } from 'lodash';
+import { PubSub } from 'graphql-subscriptions';
+import { passOrThrow, isArray, isFunction, mapOverProperties } from '../util';
+
+import { Entity } from '../entity/Entity';
+
+export const pubsub = new PubSub();
+
+export const SUBSCRIPTION_TYPE_CREATE = 'onCreate';
```

Comment:
Reused the naming convention from mutation, prefixed by `on` like an event listener does.